inst/tests/tests.Rraw

```diff
@@ -2868,7 +2868,7 @@ DT = data.table(a=1:3,b=4:6,b=7:9,c=10:12)
 test(1005, rbind(DT,DT), data.table(a=rep(1:3,2),b=rep(4:6,2),b=rep(7:9,2),c=rep(10:12,2)))
 M <- mtcars
 colnames(M)[11] <- NA
-test(1006, print(as.data.table(M), nrows=10), output="gear NA.*1: 21.0")
+test(1006, print(as.data.table(M), nrows=10), output="gear\\sNA.*1: 21.0")
```

Comment:
this test failed because my console happened not to be wide enough & `NA` ended up on the next line... I guess this test is more robust?